bikeshed/config/status.py

```diff
@@ -86,7 +86,7 @@
 noEDStatuses = ["LS", "LS-COMMIT", "LS-BRANCH", "LD", "FINDING", "DRAFT-FINDING", "DREAM", "iso/NP", "whatwg/RD"]
 
 megaGroups = {
-    "w3c": frozenset(["act-framework", "audiowg", "csswg", "dap", "fxtf", "fxtf-csswg", "geolocation", "houdini", "html", "i18n", "immersivewebwg", "mediacapture", "processcg", "ricg", "sacg", "serviceworkers", "svg", "texttracks", "uievents", "wasm", "web-bluetooth-cg", "webappsec", "webauthn", "webml", "web-payments", "webperf", "webplatform", "webrtc", "webspecs", "webvr", "wicg"]),
+    "w3c": frozenset(["act-framework", "audiowg", "csswg", "dap", "fxtf", "fxtf-csswg", "geolocation", "houdini", "html", "i18n", "immersivewebwg", "mediacapture", "processcg", "ricg", "sacg", "serviceworkers", "svg", "texttracks", "uievents", "webapps", "wasm", "web-bluetooth-cg", "webappsec", "webauthn", "webml", "web-payments", "webperf", "webplatform", "webrtc", "webspecs", "webvr", "wicg"]),
```

Comment:
*so close* to alphabetical, but still not quite right ^_^

Reply:
Heh, busted 😅